### PR TITLE
zephyr-app: Fix compiler warnings

### PR DESCRIPTION
**Pull request by finikorg**
Fixes compiler warnings with Zephyr.

### CODE REVIEW

File: src/audio/kpb.c [hunk 1]
```diff
@@ -433,7 +433,7 @@ static int kpb_prepare(struct comp_dev *dev)
 
 	if (kpb->state == KPB_STATE_RESETTING ||
 	    kpb->state == KPB_STATE_RESET_FINISHING) {
-		comp_cl_err(&comp_kpb, "kpb_prepare(): can not prepare KPB due to ongoing reset, state log %x",
+		comp_cl_err(&comp_kpb, "kpb_prepare(): can not prepare KPB due to ongoing reset, state log %llx",
```
Comment (paulstelian97) [inline src/audio/kpb.c:436]: Wait, kpb->state_log is a 64-bit value? If so this was really broken before.

Comment (paulstelian97) [reply src/audio/kpb.c:436]: (comment applies to various other 64-bit quantities -- the old logging system _cannot_ correctly print 64-bit quantities)

Comment (finikorg) [reply src/audio/kpb.c:436]: The other option is to cast to u32.

Comment (paulstelian97) [reply src/audio/kpb.c:436]: I think on the master branch where we don't have the benefit of Zephyr logging we should do this cast.

**Pull request merged**